Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -341,6 +349,36 @@ private void InitializeInternal()
             }
         }
 
+        private Transform mixedRealityPlayspace;
+
+        /// <summary>
+        /// Returns the MixedRealityPlayspace for the local player
+        /// </summary>
+        public Transform MixedRealityPlayspace
+        {
+            get {
+                AssertIsInitialized();
+                if (mixedRealityPlayspace)
+                {
+                    return mixedRealityPlayspace;
+                }
+                else
+                {
+                    mixedRealityPlayspace = new GameObject("MixedRealityPlayspace").transform;
```

Comment:
Previously, we c[hecked to see if the parent was null](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2773/files#diff-0f24968827a5d17ade2a044a8cddf8bcL43) before creating a new parent.

We should probably do that here too, otherwise we might create a second gameobject and leave the original parent elsewhere in the scene.

Is that accounted for anywhere I missed?

Reply:
It wasn't, though it was sort of by design. My intention was to follow up with another PR that handles situations like that more robustly, perhaps even giving a warning when compiling. Would you be satisfied merging this if I created a task to track better handle incorrect parents and worked on it next week?

Reply:
Went ahead and created the task whether it goes in this PR or not:
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/2850

Reply:
This is a pretty important feature to our one click setup.

I am definitely not for just merging it and revisiting it next week.